From ab123cbbffe90aefe1579d2708bae43e53e5afb4 Tue April 25 00:00:00 2001
From: Breno Leitao <breno.leitao@gmail.com>
Date: Tue, 25 April 2017 12:33:43 +0000
Subject: [PATCH] Define PATH_MAX if not defined

PATH_MAX is not being defined on ppc64le, which causes a FTBFS.
This patch check that it is defined, otherwise, defines it.

Signed-off-by: Breno Leitao <breno.leitao@gmail.com>

--- a/lib/libaudit.h
+++ b/lib/libaudit.h
@@ -637,6 +637,10 @@
 int audit_can_write(void);
 int audit_can_read(void);
 
+#if !defined(PATH_MAX)
+#define PATH_MAX 4096
+#endif
+
 #ifdef __cplusplus
 }
 #endif
